<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmPlay</class>
 <widget class="QDialog" name="FrmPlay">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSGO - Demovoice - Play</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Demo:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtName">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Status</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtStatus">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Loading demo</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Sync</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtSync">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string>0000 ms</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutBar"/>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnStop">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>63</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lblStatus">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Please wait while the data are loaded/saved.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>63</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>75</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <layout class="QGridLayout" name="layoutLoading"/>
       </item>
       <item row="3" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>75</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>63</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
